src/plugins/formulas/formulas.js

```diff
@@ -564,6 +598,27 @@ export class Formulas extends BasePlugin {
     this.hot.runHooks('afterFormulasValuesUpdate', changes);
   }
 
+  /**
+   * Validate cell with certain address.
+   *
+   * @private
+   * @param {undefined|SimpleCellAddress} cellAddress Cell coordinates.
```

Comment:
Perhaps this check should happen outside of this method to simplify its signature? Same goes for checking if `cellAddress.sheet === this.sheetId`. You could probably just re-use the logic in `onEngineValuesUpdated` instead of calling it twice.

Style issue so I won't block because of this obviously